### DISCUSSION TITLE
`@toruslab/eccrypto` library vanilla javascript import

### DISCUSSION THREAD

**Discussion by JSeam2** [Web3Auth/Web3Auth#434] [Feature Requests]
I am building out a vanilla js app and am having an import problem regarding the `@toruslab/eccrypto` library. 

Importing it as `<script src="https://cdn.jsdelivr.net/npm/@toruslabs/eccrypto"></script>` fails likely due to some library dependency issues.

Any ideas on how this can be integrated?

**Comment by JSeam2** [comment]
Found another eccrypto library that works with vanilla js. Implementation seems similar to `@toruslab/eccrypto`.
https://github.com/pedrouid/eccrypto-js/releases/tag/5.3.0

**Comment by itsinz** [comment]
@yashovardhan
